proposals/sar_backscatter.json

```diff
@@ -0,0 +1,127 @@
+{
+    "id": "sar_backscatter",
+    "summary": "Computes backscatter from SAR input",
+    "description": "Computes backscatter from SAR input.\n\nNote that backscatter computation may require instrument specific metadata that is tightly coupled to the original SAR products. As a result, this process may only work in combination with loading data from specific collections, not with general data cubes.",
+    "categories": [
+        "cubes",
+        "sar"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The source data cube containing SAR input.",
+            "schema": {
+                "subtype": "raster-cube",
+                "type": "object"
+            }
+        },
+        {
+            "name": "elevation_model",
```

Comment:
Isn't it cleaner to move parameter `elevation_model` after `orthorectify` and `rtc`?

Reply:
Depends a bit how often we think the DEM will be chosen in contract to the other parameters. I assumed choosing a DEM would be a "best-practice", then it's good where it is. But my assumption could be wrong.
Another reason was to align with the other processes in this PR. For all of them which have such an option, it's the first optional parameter.

Reply:
As far as I currently understand `elevation_model` only makes sense when `orthorectify` and/or `rtc` are enabled, so it looks more intuitive if that parameter comes after these.
Also, elevation_model has a default of null to let backend pick a model, so it is less critical to specify than `orthorectify` or `rtc`

Reply:
That sounds reasonable.

Reply:
Moved behind the two options.